src/sema/namespace.rs

```diff
@@ -1148,7 +1148,7 @@ impl Namespace {
                         .into_iter()
                         .map(|p| {
                             if let Some(name) = p.id {
-                                diagnostics.push(Diagnostic::error(
+                                diagnostics.push(Diagnostic::warning(
```

Comment:
Should success still be false here even if this is not an error anymore?

Reply:
Good point, fixed